resources/views/omnomcom/withdrawals/show.blade.php

```diff
@@ -130,73 +132,77 @@ class="btn btn-outline-success btn-block">
-                                               'text' => '(Revert)',
-                                               'title' => 'Confirm Revert',
-                                               'message' => 'Are you sure you want to revert this withdrawal? The user will <b>NOT</b> automatically receive an e-mail about this!',
-                                               'confirm' => 'Revert',
-                                            ])
+                                        @isset($data->user->bank)
+                                            <td>
+                                                {{ $data->user->bank->iban }}
+                                                <span class="text-muted">/ {{ $data->user->bank->bic }}</span>
+                                            </td>
+                                            <td>{{ $data->user->bank->machtigingid }}</td>
                                         @else
-                                            <a href="{{ route('omnomcom::withdrawal::deleteuser', ['id' => $withdrawal->id, 'user_id' => $data->user->id]) }}" class="text-white fw-bold underline-on-hover">
-                                                Remove
-                                            </a>
-
-                                            |
-
-                                            @include('components.modals.confirm-modal', [
-                                               'action' => route('omnomcom::withdrawal::markfailed', ['id' => $withdrawal->id, 'user_id' => $data->user->id]),
-                                               'text' => 'Failed',
-                                               'title' => 'Confirm Marking Failed',
-                                               'message' => 'Are you sure you want to mark this withdrawal for '.$data->user->name.' as failed? They <b>will</b> automatically receive an e-mail about this!',
-                                               'classes' => 'text-white fw-bold underline-on-hover'
-                                            ])
-
-                                            |
-
-                                            @include('components.modals.confirm-modal', [
-                                               'action' => route('omnomcom::withdrawal::markloss', ['id' => $withdrawal->id, 'user_id' => $data->user->id]),
-                                               'text' => 'Loss',
-                                               'title' => 'Confirm Marking Loss',
-                                               'message' => 'Are you sure you want to mark this withdrawal for '.$data->user->name.' as a loss? <b>This cannot easily be undone!</b>',
-                                               'classes' => 'text-white fw-bold underline-on-hover'
-                                            ])
```

Comment:
Maybe still keep the single update buttons as this is easier to use when just removing or marking one user?

Reply:
I removed them because when you do select orderlines and then remove one orderline your selection will be gone. I think only having one way is more clear.

Reply:
Yes I agree, I do not understand the confirm-modal enough yet to implement this though